lib/Doctrine/ORM/Proxy/InternalProxy.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Proxy;
+
+use Doctrine\Persistence\Proxy;
+
+/**
+ * @internal
+ *
+ * @method void __setInitialized(bool $initialized)
```

Comment:
if it is an internal API, can't we define the method in the interface directly ?

Reply:
Not possible because this overlaps with exiting declarations on legacy Proxy interfaces / implementations.